polynote-kernel/src/main/scala/polynote/kernel/remote/transport.scala

```diff
@@ -378,8 +375,12 @@ object SocketTransport {
     private def readBuffer(): Option[Option[ByteBuffer]] = incomingLengthBuffer.synchronized {
       incomingLengthBuffer.rewind()
       while(incomingLengthBuffer.hasRemaining) {
-        if(socketChannel.read(incomingLengthBuffer) == -1) {
-          return None
+        try {
+          if(socketChannel.read(incomingLengthBuffer) == -1) {
+            return None
+          }
+        } catch {
+          case err: IOException => return None
```

Comment:
Is the error already logged somewhere else?

Reply:
Hmm, I guess it wouldn't be. I think this is left over from debugging, I'll revert it.